src/VisualStudio/CSharp/Test/CSharpVisualStudioTest.csproj

```diff
@@ -103,8 +103,9 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
     <ProjectReference Include="..\..\..\Features\Core\Portable\Features.csproj">
-      <Project>{edc68a0e-c68d-4a74-91b7-bf38ec909888}</Project>
+      <Project>{EDC68A0E-C68D-4A74-91B7-BF38EC909888}</Project>
       <Name>Features</Name>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
It looks like it was a part of the original commit 1b6a83f9dcd although I'm not sure why VS updated this file since I did not modify this file manually